src/DataCollector/HttpDataCollector.php

```diff
@@ -110,11 +110,24 @@ public function getCallCount() : int
      *
      * @return integer
      */
-    public function getErrorCount() : int
+    public function getErrorCount(): int
     {
-        return count(array_filter($this->getMessages(), function (LogMessage $message) {
-            return $message->getLevel() === LogLevel::ERROR;
-        }));
+        return count($this->getErrorsByType(LogLevel::ERROR));
+    }
+
+    /**
+     * @param string $type
+     *
+     * @return array
+     */
+    public function getErrorsByType(string $type): array
+    {
+        return array_filter(
+            $this->getMessages(),
+            function (LogMessage $message) use ($type) {
+                return $message->getLevel() == $type;
```

Comment:
why not string type here? In previous version I had string type comparison:
```php
return $message->getLevel() === LogLevel::ERROR;
```

Reply:
oh, you meant strict type probably. Yes, sure you are right. It is a leftover from the commit where I did add strict type per file. Will correct it tonight.